frontend/src/metabase/query_builder/components/EditableText/EditableText.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+
+import { Root, StyledTextArea } from "./EditableText.styled";
+
+import { KEY_ESCAPE } from "metabase/lib/keyboard";
+
+type text = string | null | undefined;
+
+interface Props {
+  value: text;
+  onChange?: (val: string) => void;
+}
+
+const EditableText = ({ value, onChange }: Props) => {
+  const [memo, setMemo] = useState<text>(value);
```

Comment:
nit: Could be called `value, setValue` after changing the props to use `initialValue`